python/cudf/cudf/benchmarks/bench_cudf_io.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
+
+from cudf.tests import dataset_generator as dg
+
+import pytest
+import cudf
+import numpy as np
+import pandavro as pav
+
+common_dtypes_meta = [
```

Comment:
We can speed up the benchmark execution by grouping related types and running the benchmark on mixed tables.
One way to do this is to have `get_random_dataframe` accept a list of types, another would be to generate columns separately and combine the table (this should be cheap).

Reply:
So you want to combine ints (int8, int16, int32 ..) and make a table out of it and run benchmark on it ?